frontend/src/concepts/pipelines/content/tables/usePipelineLoadMore.ts

```diff
@@ -96,9 +96,10 @@ export const usePipelineVersionLoadMore = ({
       sortDirection,
       filter,
     });
-    setMoreData(result.versions || []);
+    showMoreData((flag) => !flag);
     dataRef.current = [...dataRef.current, ...(result.versions || [])];
     pageTokenRef.current = result.next_page_token;
+    // Return undefined to avoid type check error because there is a if return above
```

Comment:
I don't think this comment is necessary, but rather one explaining the purpose of `showMoreData` above.